Update add-use-text-secrets.md (#7332)
harness/developer-hub: docs/platform/secrets/add-use-text-secrets.md

7. Select **Save**.

  :::warning

>    Inline secrets of text types if deleted from Harness will also delete the secrets from the external secrets manager. It is recommended to always take a backup of these secrets.

Reviewer: ```suggestion
   Inline secrets of text types deleted in Harness are also deleted from the external secrets manager. Harness recommends that you always back up these secrets.
```
